Docker swarm.md

```diff
@@ -0,0 +1,15 @@
+### A Brief Overview
+
+Docker Swarm is a container orchestration tool that comes built into Docker. It allows you to manage a cluster of Docker engines as a single virtual Docker engine. This means you can deploy and manage containers across multiple machines seamlessly. Swarm handles load balancing, scaling, and ensures high availability of your services.
+
+### cute comparison between Docker Swarm and Kubernetes
+
+Both Docker Swarm and Kubernetes have their strengths and are suitable for different use cases. Docker Swarm is often favored for simpler deployments or when ease of use is a priority, while Kubernetes is preferred for more complex and scalable environments where advanced features are needed.
```

Comment:
Also docker swarm is nice because you can set your computer as a part of a swarm cluster very easially.